client/scripts/views/pages/create_community/starter_community_form.tsx

```diff
@@ -110,12 +110,14 @@ export class StarterCommunityForm implements m.ClassComponent {
                 break;
               }
             }
+            const { iconUrl } = this.state.form;
             try {
               const res = await $.post(`${app.serverUrl()}/createChain`, {
                 jwt: app.user.jwt,
                 address: '',
                 type: ChainType.Offchain,
                 network: baseToNetwork(this.state.form.base),
+                icon_url: iconUrl,
                 ...this.state.form,
```

Comment:
we should destructure the rest of these above on 113 so we don't pass in iconUrl

Reply:
Done